### ISSUE TITLE
Unable to obtain new ArchLinux image (v201704) from atlas.hashicorp.com  

### ISSUE THREAD

**Issue by proycon** [terrywang/vagrantboxes#26]
This is a follow up of #25. Thanks for so quickly releasing a fixed image. However, there seems to be a problem obtaining it from atlas.hashicorp.com (404):

```
$ vagrant box update                                                                                                                                                                                    
==> default: Checking for updates to 'terrywang/archlinux'
    default: Latest installed version: 1.0.0
    default: Version constraints:
    default: Provider: virtualbox
==> default: Updating 'terrywang/archlinux' with provider 'virtualbox' from version
==> default: '1.0.0' to '201704'...
==> default: Loading metadata for box 'https://atlas.hashicorp.com/terrywang/archlinux'
==> default: Adding box 'terrywang/archlinux' (v201704) for provider: virtualbox
    default: Downloading: https://atlas.hashicorp.com/terrywang/boxes/archlinux/versions/201704/providers/virtualbox.box
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.

The requested URL returned error: 404 Not Found
```

Same thing if I delete it manually and re-add instead of update:

```
$ vagrant box add terrywang/archlinux                                                                                                                                                                  
==> box: Loading metadata for box 'terrywang/archlinux'
    box: URL: https://atlas.hashicorp.com/terrywang/archlinux
==> box: Adding box 'terrywang/archlinux' (v201704) for provider: virtualbox
    box: Downloading: https://atlas.hashicorp.com/terrywang/boxes/archlinux/versions/201704/providers/virtualbox.box
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.

The requested URL returned error: 404 Not Found
```

Is there something I'm missing or did it not get published correctly?

**Comment by mtpereira** [comment]
Hi,

I'm getting the exact same error. If you access that URL ob a browser you get the same 404.

**Comment by proycon** [comment]
Issue seems fixed now with a new release (v3.0.4), thanks!
